tools/build-webview

```diff
@@ -60,6 +37,8 @@ absolutify() {
 ROOT="$(git rev-parse --show-toplevel)"
 cd "$ROOT"
 
+. "$ROOT"/tools/lib/ensure-coreutils.sh
```

Comment:
This could use a comment, I think.

Reply:
Could do, but: what would you like a comment to say? The obvious thing that comes to my mind is what the name already aims to say :smile: : "ensure we have coreutils".

For details in turn of what *that* means, I think it's going to be hard to compete in this spot with the comment at the top of that file itself.

Reply:
I'd been thinking something along the lines of "Ensure we have `readlink`" (rather than the moderately nebulous "coreutils").

Reply:
> I'd been thinking something along the lines of "Ensure we have `readlink`" (rather than the moderately nebulous "coreutils").

Sure, that'd be reasonable.

OTOH now that we have a version of this we believe we *can* just count on for all dev scripts, I think it could be helpful to go ahead and add this to the top of basically all of them (that are in shell). Effectively that makes it part of our shell-scripting environment: we just write all our shell scripts in "Bash, with GNU coreutils". Which is a helpful simplification for thinking about editing any of these scripts -- if you need `readlink -f` or whatever, no need to consider whether this is one of the scripts that has coreutils or doesn't, just do it because they all have it.